package.json

```diff
@@ -10,27 +10,22 @@
   "scripts": {
     "lint": "npm run lint:eslint",
     "lint:eslint": "eslint . --ext .js,.html",
-    "start": "es-dev-server --app-index demo/index.html --node-resolve --dedupe --open --watch",
```

Comment:
There was never a demo page... I think this was just leftover from using the init template.